bedrock/externalpages/templates/externalpages/pocket/add.html

```diff
@@ -0,0 +1,116 @@
+               <li class="app-store-badge"><a href="https://play.google.com/store/apps/details?id=com.ideashower.readitlater.pro" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Google Play Store</a></li>
+               <li class="app-store-badge"><a href="https://www.amazon.com/dp/B0057PAY8G" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Amazon Store</a></li>
+            </ul>
+            <ul class="device-list">
+               <li><a href="https://chrome.google.com/webstore/detail/save-to-pocket/niloccemoadcdkdjlinkgdfekeahmflj?hl=en" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Windows <span class="rarrow">›</span></a></li>
+               <li><a href="https://help.getpocket.com/article/1006-getting-started-with-kobo" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Kobo e-reader <span class="rarrow">›</span></a></li>
+               <li><a href="https://getpocket.com/apps/windowsphone/" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Windows Mobile <span class="rarrow">›</span></a></li>
+               <li><a href="https://getpocket.com/apps/blackberry/" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Blackberry <span class="rarrow">›</span></a></li>
+               <li><a href="https://getpocket.com/apps/" target="_blank" rel="noopener noreferrer" class="dark-inline-link">More Apps <span class="rarrow">›</span></a></li>
+            </ul>
+         </div>
+      </div>
+   </section>
+
+   <section class="add-save">
+      <div class="add-save__wrapper">
+         <div class="add-save__title">
+            <h2>Unlimited Ways to Save</h2>
+         </div>
+         <div class="add-save__context">
+            <div class="add-save__context-email">
+               <h3>Save via e-mail</h3>
+               <p>email any link to your list by sending it to <a href="mailto:add@getpocket.com" class="dark-inline-link">add@getpocket.com</a> </p>
+               <div class="add-save__context-image">
+                  <img src="{{ static('img/externalpages/pocket/save-via-email.png') }}" alt="">
+               </div>
+               <a
+                  href="https://help.getpocket.com/article/1020-saving-to-pocket-via-email" target="_blank" rel="noopener noreferrer" class="dark-inline-link"
+                  >Learn more</a
+                  >
+            </div>
+            <div class="add-save__context-apps">
+               <h3>Integrated in 1500+ Apps</h3>
+               <p>Save to Pocket using your favorite apps, such as:</p>
+               <div class="app-types">
+                  <ul class="app-board">
+                     <li>app</li>
+                     <li>app</li>
+                     <li>app</li>
+                     <li>app</li>
+                     <li>app</li>
+                     <li>app</li>
```

Comment:
Is this unfinished content?

![image](https://user-images.githubusercontent.com/400117/143253540-0d274a10-3a86-4984-8503-5896a23fc7b0.png)

Reply:
The way they added the icons to https://getpocket.com/add is super weird, where they had all the icons be one background image for each section and then adding overlays and links on each icon. I couldn't find good icons to replace that with (yet) so i made placeholder squares for where the icons should be placed.

Reply:
I think using a CSS sprite-map is ok, if we can reuse what they already did here? Else we should file a followup issue to fix these? As a side note, repeating `app` for every item in the list isn't great for accessibility.

Reply:
Oh yeah it definitely wasn't a final go at the page, i just wanted to get the draft of it up and running. I also still need to fix its mobile view. It was my first go at getting a page up for Pocket, so gotta clean a lot here. I also didn't know that CSS sprite-maps were a thing (TIL) ! I'll clean up the page shortly. Thanks for your review :)